build.js

```diff
@@ -53,6 +53,27 @@ function createModule() {
     const esPath = `./es/${s.slice(2)}.js`;
     const key = s.endsWith("/index") ? s.slice(0, -6) : s;
     if (!key) return;
+
+    // create subpackage package.json
+    if (key !== ".") {
+      const subPackagePath = path.join(OUT_DIR, key);
+      if (!fs.existsSync(subPackagePath)) {
+        fs.mkdirSync(subPackagePath, { recursive: true });
+      }
+      const subPackageJson = {
+        "main": path.relative(subPackagePath, path.join(OUT_DIR, libPath)),
+        "module": path.relative(subPackagePath, path.join(OUT_DIR, esPath)),
+        "es2015": path.relative(subPackagePath, path.join(OUT_DIR, esPath)),
+        "jsnext:main": path.relative(subPackagePath, path.join(OUT_DIR, esPath)),
+        "types": path.relative(subPackagePath, path.join(OUT_DIR, typesPath)),
+        "sideEffects": false
```

Comment:
This should be `key.includes("init/")`  to account for `import "mingo/init/..."` which has side effects.

Reply:
@kofrasa  you are right. Fixed in ed15fdade307faa505246fb6eb325676e10461de